packages/perseus-core/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@khanacademy/perseus-core",
+    "description": "Shared Perseus infrastructure",
+    "author": "Khan Academy",
+    "license": "MIT",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Khan/perseus.git"
+    },
+    "bugs": {
+        "url": "https://github.com/Khan/perseus/issues"
+    },
+    "module": "dist/es/index.js",
+    "main": "dist/index.js",
+    "source": "src/index.ts",
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
+    },
+    "dependencies": {
+        "@khanacademy/event-schemas": "git+ssh://git@github.com:Khan/event-schemas.git#v162.0.0"
```

Comment:
Because we introduced perseus-core, we can have a single place that we upgrade the event-schema now (instead of once in `math-input` and another in `perseus`.

Reply:
Ouch, good call!  I agree that we can simply create our own analytics types in Perseus and map them in webapp without much extra hassle. I'm switching to that direction.